addons/sale/data/mail_data.xml

```diff
@@ -107,5 +107,103 @@
             <field name="user_signature" eval="False"/>
             <field name="auto_delete" eval="True"/>
         </record>
+
+        <record id="order_confirmation_mail_template" model="mail.template">
+            <field name="name">Order confirmation e-mail</field>
+            <field name="model_id" ref="sale.model_sale_order"/>
+            <field name="email_from">${object.user_id.email_formatted |safe}</field>
+            <field name="email_to">${(object.partner_id.email or '')|safe}</field>
+            <field name="subject">Your order ${object.name}</field>
+            <field name="body_html" type="xml">
+<div style="margin: 0px; padding: 0px;">
+    <p style="margin: 0px; padding: 0px; font-size: 13px;">
+        % set transaction = ctx.get('transaction')
+        Hello,<br/><br/>
+        Your order ${object.name} amounting in ${object.currency_id.symbol}${object.amount_total}
+        % if transaction and transaction.state != 'pending':
+            has been confirmed.
+        % else:
+            will be validated when the payment is received.
+        % endif
+    </p><br/>
+    % if transaction and transaction.state == 'pending':
+        <strong>PAYMENT INSTRUCTIONS</strong><br/>
+        ${transaction.acquirer_id.post_msg | safe}<br/>
+        Communication: ${object.reference}<br/><br/>
+    % endif
+    <a href="${ctx.get('access_url')}" target="_blank" style="background-color: #44a19e; padding: 8px 16px; text-decoration: none; color: #fff; border-radius: 5px;">View Your Order</a>
```

Comment:
Why not order._get_share_url ?

Reply:
@tde-banana-odoo , Private methods are not accessible in Jinja templates. Thanks